backend/hitas/models/__init__.py

```diff
@@ -14,9 +14,9 @@
 )
 from hitas.models.indices import (
     ConstructionPriceIndex,
-    ConstructionPriceIndexPre2005,
+    ConstructionPriceIndex2005Equal100,
     MarketPriceIndex,
-    MarketPriceIndexPre2005,
+    MarketPriceIndex2005Equal100,
```

Comment:
I don't really understand, what's the reasoning behind the naming change? (More specifically the `Equal100` part)

Reply:
There's two indices for market price index:
- `Markkinahintaindeksi` is used for apartments constructed before *2011* (https://www.hel.fi/static/kv/asunto-osasto/hitas-markkinahintaindeksi.pdf)
 - `Markkinahintaindeksi 2005=100` is used for apartments constructed *2011* onwards (https://www.hel.fi/static/kv/asunto-osasto/hitas-indeksit-2005-100.pdf)
 
 The `pre 2005` part is wrong. It could be a `pre 2011` but looking at the https://www.hel.fi/static/kv/asunto-osasto/hitas-indeksit-2005-100.pdf the `2005-100` is part of the filename and it's named `Markkinahintaindeksi 2005=100` (meaning index was 100 in 2005). I did not want to invent a new name, I believe that would be more confusing :)

  -

Reply:
All right, that part of the naming scheme wasn't clear from the commits, good to know :)